fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1418,7 +1503,7 @@ module.exports = function(config, DB) {
               })
               .then(function () {
                 // Attempt to verify session token
-                return db.verifyToken(SESSION_TOKEN.tokenVerificationId, { uid: uid })
+                return db.verifyTokens(SESSION_TOKEN.tokenVerificationId, { uid: uid })
```

Comment:
Do we need an explicit test to check that you can verify *both* a sessionToken and a keyFetchToken with a single call to `verifyToens` using the same tokenVerificationId?  Or is that covered/implied by  other tests?